app/index.html

```diff
@@ -6,6 +6,7 @@
     <title>My Site | Real Dev Squad</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="/icons/favicon.ico" />
```

Comment:
Does the `favicon.ico` file exist? or even the icons folder exists in the code?

Reply:
@rohan09-raj Yes, I have added the favicon.ico and the icons folder

Reply:
Ohh ok, sorry I missed it. Thanks I can see it now